lib/planner.go

```diff
@@ -60,6 +60,10 @@ func CreatePlan(ctx context.Context, inputPath, outputPath string, customization
 	}
 	logrus.Infoln("Configuration loading done")
 
+	if len(p.Spec.Services) == 0 && len(p.Spec.InvokedByDefaultTransformers) == 0 {
```

Comment:
Won't the list of services only become available only after the planning is done below?

Reply:
fixed it 😄